webapp/app.py

```diff
@@ -517,6 +521,18 @@ async def timing_middleware(request, handler, **kwargs):
 
         return res
 
+    @web.middleware
+    async def gzip_middleware(request, handler, **kwargs):
+        """ Middleware that compress response using gzip"""
+        res = await handler(request, **kwargs)
+        LOGGER.info(request.headers)
```

Comment:
make it LOGGER.debug please, info is default level here and logs would be overly verbose

Reply:
oh, sorry, that's forgotten log, removing it